include/swift/AST/Decl.h

```diff
@@ -374,6 +374,10 @@ bool conflicting(ASTContext &ctx,
                  bool *wouldConflictInSwift5 = nullptr,
                  bool skipProtocolExtensionCheck = false);
 
+enum class SpecialDistributedProperty {
```

Comment:
```suggestion
enum class SpecialDistributedActorProperty {
```

Just to that it's clearer that we mean specifically properties of a distributed actor; (a "distributed property" exists and is `distributed var {}` but that's different)